core/src/main/java/org/stellar/anchor/util/TransactionMapper.java

```diff
@@ -137,6 +137,7 @@ public static GetTransactionResponse toGetTransactionResponse(
         .sep(PlatformTransactionData.Sep.SEP_6)
         .kind(PlatformTransactionData.Kind.from(txn.getKind()))
         .status(SepTransactionStatus.from(txn.getStatus()))
+        .fundingMethod(txn.getType())
```

Comment:
Should this be `getFundingMethod`?

Reply:
This is sepTxn from db. It'a still saved as `type`

Reply:
I thought so but wasn't sure because on line 98, `fundingMethod` is set to `txn.getFundingMethod()`.

Reply:
That's for SEP-31, which doesn't have `type` originally, and I added `fundingMethod `in https://github.com/stellar/java-stellar-anchor-sdk/pull/1581